File: 03-MobileTecnico-Android/app/src/main/java/com/helpdesk/tecnico/models/LoginResponse.java
package com.helpdesk.tecnico.models;

public class LoginResponse {
    private boolean sucesso;
    private String token;
    private String mensagem;

    public boolean isSucesso() { return sucesso; }
    public void setSucesso(boolean sucesso) { this.sucesso = sucesso; }

    public String getToken() { return token; }
    public void setToken(String token) { this.token = token; }

    public String getMensagem() { return mensagem; }
    public void setMensagem(String mensagem) { this.mensagem = mensagem; }
}

File: 02-DesktopManager-WinForms/Models/LoginResponse.cs
using DesktopManager.Models;

namespace DesktopManager.Models
{
    public class LoginResponse
    {
        public bool Sucesso { get; set; }
        public string Token { get; set; } = string.Empty;
        public string Mensagem { get; set; } = string.Empty;
        public Usuario? Usuario { get; set; }
    }
}